### PR TITLE
Support service setup for multiple container

### PR DESCRIPTION
**Pull request by arjav-desai**
# Description

Support service creation for multiple containers defined in helidon workload

Fixes VZ-2593

# Checklist 

As the author of this PR, I have:

- [x] Checked that I included or updated copyright and license notices in all files that I altered
- [x] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: application-operator/controllers/helidonworkload/helidonworkload_controller_test.go [hunk 1]
```diff
@@ -7,6 +7,8 @@ import (
 	"bufio"
 	"context"
 	"io/ioutil"
+	"k8s.io/apimachinery/pkg/util/intstr"
```
Comment (kminder) [inline application-operator/controllers/helidonworkload/helidonworkload_controller_test.go:10]: Import ordering

File: application-operator/controllers/helidonworkload/helidonworkload_controller.go [hunk 2]
```diff
@@ -236,18 +237,25 @@ func (r *Reconciler) createServiceFromDeployment(workload *vzapi.VerrazzanoHelid
...
-		// short-term in the case that ports are modified after creation.
-		if len(deploy.Spec.Template.Spec.Containers[0].Ports) > 0 {
-			s.Spec.Ports = []corev1.ServicePort{
-				{
-					Name:       deploy.GetName(),
-					Port:       deploy.Spec.Template.Spec.Containers[0].Ports[0].ContainerPort,
-					TargetPort: intstr.FromInt(int(deploy.Spec.Template.Spec.Containers[0].Ports[0].ContainerPort)),
-					Protocol:   corev1.ProtocolTCP,
-				},
+		for _, container := range deploy.Spec.Template.Spec.Containers {
```
Comment (kminder) [inline application-operator/controllers/helidonworkload/helidonworkload_controller.go:240]: As mentioned in the original comment there is a garbage collection issue.  This is only relevant if the controller supports update.  Does it?

Comment (arjav-desai) [reply application-operator/controllers/helidonworkload/helidonworkload_controller.go:240]: That comment was carried over from OAM's containerized workload code. We call Patch on Reconcile in this controller which supports update i.e. equivalent to CreateOrUpdate call.

Also, I am a update to port would required, user to first update the component and the app config; so it should not be "short-term" wrt to updating ports.

File: application-operator/controllers/helidonworkload/helidonworkload_controller_test.go (hunk 1 above)
Comment (arjav-desai) [reply application-operator/controllers/helidonworkload/helidonworkload_controller_test.go:10]: Have to fix my Idea cause I ran "go fmt" and it arranges this way. Fixed manually for now.

**Review by kminder — APPROVED** [review]
Approved.

**Review by fredtibbitts — APPROVED** [review]

**Pull request merged**